test/integration/configurations/nighthawk_track_timings.yaml

```diff
@@ -0,0 +1,34 @@
+admin:
```

Comment:
Can we add a comment at the top of this, explaining its purpose in just a sentence, and draw attention to the relevant/unique part of the configuration (which I think is the time-tracking filter)